content/client-lib-development-guide/features.textile

```diff
@@ -1348,7 +1365,7 @@ h4. ChannelDetails
 
 h4. ChannelStatus
 
-* @(CHS1)@ @ChannelStatus@ is a type that contains status and occupancy for a channel 
+* @(CHS1)@ @ChannelStatus@ is a type that contains status and occupancy for a channel
```

Comment:
These changes are whitespace removal that presumably has been done automatically by the editor and they're not intended to be part of this change. They should usually be in a separate commit.

Reply:
0ab5af22c5284c4e17e6324413a2207af54a393a